evrMrmApp/src/drvem.cpp

```diff
@@ -285,11 +285,6 @@ try{
 
     } else if(formfactor==formFactor_mTCA) {
 
-        // mapping to TCLKA and TCLKB as UNIV16, 17
-        // we move down to UNIV0, 1
-        outputs[std::make_pair(OutputFPUniv,0)]=new MRMOutput(SB()<<n<<":FrontUnivOut0", this, OutputFPUniv, 16);
-        outputs[std::make_pair(OutputFPUniv,1)]=new MRMOutput(SB()<<n<<":FrontUnivOut1", this, OutputFPUniv, 17);
-
```

Comment:
If this is removed, where is the configuration of TCLKA and TCLKB done on the MTCA EVR?

Reply:
I found the solution to my answer above in the pull request #95 and commit 64b576325f4c8771b07360b42ee5d721a6a326ac, where the number of UNIV-IOs is extended to 18 and the TCLKA and TCLKB are mapped as UNIV-IO16 and 17.

Now I only have a question how this will be handled on the GUI.

Reply:
```
$(P)Out-TCLKA-Pat-Low00_15-SP etc.
```